code/wdb/datastripe.q

```diff
@@ -42,12 +40,15 @@ initdatastripe:{
     // update endofperiod function
     endofperiod::.wdb.datastripeendofperiod;
     
-    .wdb.tablekeycols:.ds.loadtablekeycols[];  // replace with dictionaries from json file
+    // load in variables
+    .wdb.tablekeycols:.ds.loadtablekeycols[];
     t:tables[`.] except .wdb.ignorelist;
+
+    // create or load the access table
     .wdb.access: @[get;(` sv(.ds.td;.proc.procname;`access));([] table:t ; start:0Np ; end:0Np ; stptime:0Np ; keycol:`sym^.wdb.tablekeycols[t])];
     modaccess[.wdb.access];
     (` sv(.ds.td;.proc.procname;`access)) set .wdb.access;
-    .wdb.access:{[x] last .wdb.access where .wdb.access[`table]=x} each (key .wdb.tablekeycols);
+    .wdb.access:{[x] last .wdb.access where .wdb.access[`table]=x} each t;
     .wdb.access:`table xkey .wdb.access;
```

Comment:
I might be missing something but I _think_ this is the same:

```suggestion
    .wdb.access:select by table from .wdb.access where table in t;
```

Reply:
That does appear more concise & coherent - I can commit this change, if no-one else has any objections?